packages/backend/src/services/ProjectService/ProjectService.test.ts

```diff
@@ -0,0 +1,41 @@
+import { projectModel } from '../../models/models';
+import { ProjectService } from './ProjectService';
+import {
+    expectedSqlResults,
+    projectAdapterMock,
+    user,
+} from './ProjectService.mock';
+import { analytics } from '../../analytics/client';
+
+jest.mock('../../analytics/client', () => ({
+    analytics: {
+        track: jest.fn(),
+    },
+}));
+
+jest.mock('../../models/models', () => ({
+    projectModel: {},
+}));
+
+describe('DashboardService', () => {
```

Comment:
The test names are related to dashboards tests

Reply:
```suggestion
describe('ProjectService', () => {
```